File: POS System - Retaurant - 2022/POS System/MsgStatus.cs
using System;
using System.Windows.Forms;

namespace POS_System
{

    public partial class MsgStatus
    {

        private void Timer1_Tick(object sender, EventArgs e)
        {
            Close();
        }

        private void MsgStatus_KeyDown(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.Escape)
                Close();
        }

        private void frmStatus_Load(object sender, EventArgs e)
        {
            FormBorderStyle = FormBorderStyle.None;
            Label1.Text = mdlGeneral.gblMessage;
        }

    }
}

File: POS System - Retaurant/POS System/MsgStatus.vb

Public Class MsgStatus

    Private Sub Timer1_Tick(ByVal sender As Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        Me.Close()
    End Sub

    Private Sub MsgStatus_KeyDown(ByVal sender As Object, ByVal e As System.Windows.Forms.KeyEventArgs) Handles Me.KeyDown
        If e.KeyCode = Keys.Escape Then Me.Close()
    End Sub

    Private Sub frmStatus_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
        Me.FormBorderStyle = Windows.Forms.FormBorderStyle.None
        Label1.Text = gblMessage
    End Sub

End Class